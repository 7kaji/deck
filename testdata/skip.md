# Skip

---

# Skip this Page

<!-- {"skip": true} -->

---

# Hello
