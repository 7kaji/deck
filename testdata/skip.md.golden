[
  {
    "layout": "",
    "titles": [
      "Skip"
    ],
    "headings": {
      "1": [
        "Skip"
      ]
    }
  },
  {
    "layout": "",
    "skip": true,
    "titles": [
      "Skip this Page"
    ],
    "headings": {
      "1": [
        "Skip this Page"
      ]
    }
  },
  {
    "layout": "",
    "titles": [
      "Hello"
    ],
    "headings": {
      "1": [
        "Hello"
      ]
    }
  }
]